CHANGES.rst

```diff
@@ -8,6 +8,11 @@
     own ``.luacheckrc`` detection. Therefore ``flycheck-luacheckrc`` is
     no longer used [GH-1057]
 
+- New features:
```

Comment:
Thanks for updating the changelog, @c-nixon.  However, please move the "New features" item *below* the "new syntax checkers" item.  We think that new syntax checkers are more important as the extend the languages Flycheck supports.